docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,321 @@
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   ./viam-server --appimage-extract-and-run -config /etc/viam.json
+   ```
+
+   {{< /alert >}}
+
+1. Once complete, verify that step 3 on the **Setup** tab indicates that your robot has successfully connected.
+
+## Generate your robot configuration
+
+When you built the C++ SDK, the build process also built the `example_audio_classification_client` binary, which includes a `--generate` function that determines and creates the necessary robot configuration to support this example.
+
+To generate your robot's configuration using `example_audio_classification_client`:
+
+1. First, determine the full path to the `yamnet/classification` model you just downloaded.
+   If you followed the instructions above, this path is: <file>~/example_workspace/lite-model_yamnet_classification_tflite_1.tflite</file>.
+
+1. Next, determine the full path to the `example_mlmodelservice_tflite` modular resource example provided with the Viam C++ SDK.
+   If you followed the instructions above, this path is: <file>~/example_workspace/opt/bin/example_mlmodelservice_tflite</file>.
+
+1. Run the `example_audio_classification_client` binary, proving both paths to the `--generate` function in the following fashion:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cd ~/example_workspace/opt/bin
+   ./example_audio_classification_client --generate --model-path ~/example_workspace/lite-model_yamnet_classification_tflite_1.tflite --tflite-module-path ~/example_workspace/opt/bin/example_mlmodelservice_tflite > ~/example_workspace/viam-example-mlmodel-config.json
+   ```
+
+1. Verify that the resulting configuration file was created successfully:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cat ~/example_workspace/viam-example-mlmodel-config.json
+   ```
+
+1. Copy the contents of this file.
+   Then return to your robot's page on [the Viam app](https://app.viam.com), select the **Config** tab, select **Raw JSON**, and paste the configuration into the text area.
+
+1. Click the **Save config** button at the bottom of the page.
+   Now, when you switch back to **Builder** mode, you can see the new configuration settings under the **Services** and **Modules** subtabs.
+
+This generated configuration features the minimum required configuration to support this tutorial: `services` parameters for the [ML model](/services/ml/) service and `modules` parameters for the `example_mlmodelservice_tflite` module.
+
+## Run the inference client
+
+With everything configured and running, you can now run the inference client that connects to `viam-server` and uses the `example_mlmodelservice_tflite` module.
+
+1. First, determine your robot address and location secret. To do so, navigate to [the Viam app](https://app.viam.com), select the **Code sample** tab, and toggle **Include secret**.
```

Comment:
I don't particularly like the workflow of visiting the code samples because it requires digging the secret out of the middle of the code. You can also get the location secret code from the top-level location page, by opening the "Location secret keys" card. I think that might be a better workflow to document here.

Reply:
@andf-viam - I think this is the only thing left that I think should be improved in this tutorial.